README.md

```diff
@@ -4,80 +4,93 @@
 
 Write a function that takes in a number as input and recursively finds the sum of all numbers up to and including that number.
 
-```js
-input: 6
-output: 21
+```
+function sum(n) {
+    return n && n  + sum(n - 1);
+}
 
-//21 = 6 + 5 + 4 + 3 + 2 + 1
 ```
 
 
 - ### Multiply array
 
 Write a function called `multArr` that takes in an array of numbers as an argument and recursively multiplies together all of the values in the array.
 
-```js
-multArr([2, 3, 5]); // returns 30
-multArr([5, 5, 1, 2]); //returns 50
+```
+const multArr = (arr) => {
+    if(arr.length === 0) return 1
+    let lastNum = arr.pop()
+    return multArr(arr) * lastNum
+}
+
+
 ```
 
 - ### Concatenate array
 
 Write a function called `concatArr` that takes in an array of strings as an argument and recursively concatenates the strings together into a single string, with spaces between each original string.
 
-```js
-concatArr(['is', 'it', 'tomorrow']); // returns 'is it tomorrow'
-concatArr(['or', 'just', 'the', 'end', 'of', 'time']); //returns 'or just the end of time'
+```
+
+const concatArr = (str) => {
```

Comment:
this takes in a `str` instead of `arr` as the argument.

You should also invoke the method shift otherwise it doesn't do anything. 

Also look into seeing where u can add spaces between the words that you concat